build.rs

```diff
@@ -1,57 +1,57 @@
+use git2::Repository;
 use std::env;
-use std::path::Path;
 use std::fs::{self, File};
 use std::io::Write;
-use git2::Repository;
-
+use std::path::Path;
```

Comment:
There's no need to reorder the imports. I like dependency imports being separate from std imports anyway.